core/modules/utils/dom/dragndrop.js

```diff
@@ -54,7 +56,28 @@ exports.makeDraggable = function(options) {
 				$tw.utils.addClass(event.target,"tc-dragging");
 				// Invoke drag-start actions if given
 				if(startActions !== undefined) {
-					options.widget.invokeActionString(startActions,options.widget,event,{actionTiddler: titleString});
+					// Collect our variables
```

Comment:
Is there any scope for sharing code with the eventcatcher widget?

Reply:
I don't know, the variables here are not exactly the same as the ones collected in the eventcatcher widget. The eventcatcher collects more variables...

Reply:
If I manage to share the code with the eventcatcher widget, should it be a new method in utils/dom/dom.js?

Reply:
Hi @Jermolene , now I've created a `collectDOMVariables` method in dom.js so that code can be shared between the eventcatcher, the draggable startactions and endactions